src/frontend/src/player.ts

```diff
@@ -67,17 +67,18 @@ export class Player {
     Object.entries(this._ipcEvents).forEach(event => ipcMain.on(event[0], event[1]))
   }
 
-  initStore(): Store<IPlayerStore> {
+  initStore(playerConfig: IPlayerStore): void {
     const shouldPersist = !storeManager.exists('player')
-    const store = storeManager.createStore<IPlayerStore>('player', DEFAULT_PLAYER_STORE)
 
-    if (shouldPersist) {
-      store.persist()
-    } else if (store.get('version') !== app.getVersion()) {
-      store.reset(DEFAULT_PLAYER_STORE)
+    if (storeManager.exists('player')) {
+      this.store.bulkSet(playerConfig)
```

Comment:
There is no check to the values applied (both here and in flowrStore), which means that no matter the defaults they will be overridden by `null` values at first start if the configuration has not been done entirely (which will happen 90% of the time). We need to have a fallback mechanism to the default values if the ones provided by flowr-frontend are not defined.

Reply:
fixed by Dani